### PR TITLE
updateJustification: Validators justifying the epoch boundary block at the start of the current epoch

### PR DESCRIPTION
**Pull request by dangerousfood**
## PR Description
Method implementation for [validators justifying the epoch boundary block at the start of the current epoch](https://github.com/ethereum/eth2.0-specs/blob/master/specs/core/0_beacon-chain.md#helpers).

### CODE REVIEW

**Comment by dangerousfood** [comment]
- [ ] get_current_epoch_boundary_attestations
- [ ] get_attestation_participants
- [ ] cell_div8

File: ethereum/beaconchain/src/main/java/tech/pegasys/artemis/state/StateTransition.java [hunk 1]
```diff
@@ -38,9 +37,13 @@ public void initiate(BeaconState state, BeaconBlock block) {
     }
 
     // per-epoch processing
-    if (state.getSlot() % Constants.EPOCH_LENGTH == 0) {
+    // if (state.getSlot() % Constants.EPOCH_LENGTH == 0) {
```
Comment (jrhea) [inline ethereum/beaconchain/src/main/java/tech/pegasys/artemis/state/StateTransition.java]: You should leave this uncommented.  Code that other people are working on need the epoch processing to only happen every 64 slots.

**Review by jrhea — CHANGES_REQUESTED** [review]
Just one request to uncomment the conditional that makes epoch processing happen every 64 slots.  That shouldn't be checked in as it is specific to your debugging and development of epoch processing.

**Review by jrhea — APPROVED** [review]
lgtm

**Pull request merged**